scripts/terra_service_banner.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+from google.cloud import storage as gcs
+
+
+def convert_json_to_string(json, env):
+    """Convert json file to string if supplied with custom banner json file."""
+
+    # open json file and read contents to a string
+    with open(json, "r") as j:
+        custom_banner_string = j.read()
+
+    print(f'Starting custom banner upload to {env} Terra.')
+    return(custom_banner_string)
+
+
+def update_service_banner(json, env):
```

Comment:
you could make the `json` input optional (so `update_service_banner(env, json=None)` and then if json is None, make it the default text - that would eliminate the need for `post_standard_banner` entirely

Reply:
cool, i combined the two and added print statements into the if/else block depending on the json_string that is passed in, if any.